### PR TITLE
Merge develop in main branch

### PR DESCRIPTION
**Pull request by MajdT51**
develop in main after fixing conflicts

### CODE REVIEW

File: src/main/kotlin/org/eclipse/tractusx/managedidentitywallets/models/ssi/RegisterNymPublicDto.kt [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/********************************************************************************
```
Comment (jpbu) [inline src/main/kotlin/org/eclipse/tractusx/managedidentitywallets/models/ssi/RegisterNymPublicDto.kt:1]: This file looks like a leftover, do we still need it?

**Review by jpbu — APPROVED** [review]
I briefly checked the changes and it looks correct, thanks.

Just one comment regarding a file that I think is no longer needed, however this has been overlooked in develop already and we should first fix it there to not again cause a mismatch between develop and main.

File: src/main/kotlin/org/eclipse/tractusx/managedidentitywallets/models/ssi/RegisterNymPublicDto.kt (hunk 1 above)
Comment (MajdT51) [reply src/main/kotlin/org/eclipse/tractusx/managedidentitywallets/models/ssi/RegisterNymPublicDto.kt]: done

**Comment by MajdT51** [comment]
@carslen could you please merge this PR :) thanks

File: .env.docker [hunk 2]
```diff
@@ -14,9 +17,11 @@ CX_AUTH_REDIRECT_URL=http://localhost:8080/callback
...
 
-ACAPY_API_ADMIN_URL=http://localhost:11000
 ACAPY_NETWORK_IDENTIFIER=local:test
-ACAPY_ADMIN_API_KEY=Hj23iQUsstG!dde
+ACAPY_BASE_WALLET_API_ADMIN_URL=http://localhost:11000
+ACAPY_BASE_WALLET_ADMIN_API_KEY=Hj23iQUsstGXdde
+ACAPY_API_ADMIN_URL=http://localhost:11003
+ACAPY_ADMIN_API_KEY=Hj23iQUsstGXdde
 
 BPDM_DATAPOOL_URL=https://catenax-bpdm-int.demo.catena-x.net
```
Comment (carslen) [inline .env.docker:26]: Does this file contain sensitive data in Line 5, 8, 22 and 24? If yes, you should consider these data as compromised and needs to be revoked and set to new values without publishing them again! Hard coding secrets in public repositories is an absolute no-go!

Also hard coded URLs to consortia URLs seems to be wrong (Line 26)

**Review by carslen — CHANGES_REQUESTED** [review]
Please consider much much smaller PRs in future. This will enable yourself and committers to review your code/ Pullrequest way easier. This will also help to fasten process of getting a committer for your product.

In future I won't accept this kind of PRs anymore.

Please clarify on comment regarding sensitive data in file `.env.docker` (same problems seen in file `.env.example` but I added no comment there).

File: .env.docker (hunk 2 above)
Comment (MajdT51) [reply .env.docker:26]: Thanks. No, it does not contain any sensitive data. Those values (and the values in env.example) are given only as an example and are not used in any environment.

Comment (carslen) [reply .env.docker:26]: Is it somewhere in documentation mentioned to change the content of these example key contents? Readme.md explains the purpose, but not to replace the content.

Also If you check Catena-X Pre-Prod Environment, you can find the Keys in "production": 

```shell
[~/projects/cat-x]$ k config current-context
cx-pre-prod-aks
[~/projects/cat-x]$ k -n product-managed-identity-wallets \
    get secrets catenax-managed-identity-wallets-acapy-secrets -o yaml | \
    yq '.data.acapy-admin-api-key' | base64 -d
Hj23iQUsXXXXX
[~/projects/cat-x]$ k -n product-managed-identity-wallets \
    exec pods/catenax-managed-identity-wallets-67f7975cdf-7lcrk \
    --container catenax-managed-identity-wallets -- \
    env | grep -i key
ACAPY_ADMIN_API_KEY=Hj23iQUsXXXXX
```

From my point of view there is a high chance that vulnerable installations will be available with current setup.

Comment (MajdT51) [reply .env.docker:26]: The values are similar but not equal. But yes, you are right this is not well documented and the given values could influence the chosen API-Key and make it insecure. I changed it in .env.example and .env.docker to "ApiKeySecretPlaceholder..." and we will make sure to improve those things in the future.

Comment (MajdT51) [reply .env.docker:26]: I changed also the JDBC-URL. I hope everything is okay now and it can be merged. Thanks for your time @carslen

**Review by carslen — APPROVED** [review]
LGTM

**Pull request merged**